checkov/common/bridgecrew/platform_integration.py

```diff
@@ -230,13 +233,29 @@ def is_integration_configured(self):
         """
         return self.platform_integration_configured
 
+    def set_run_details(self) -> None:
+        if self.run_temp_id:
+            request = self.http.request("GET", f'{self.run_id_url}?repositoryId={self.repo_id}&pr={self.run_temp_id}&branch={self.repo_branch}',
+                                        headers=merge_dicts({
+                                            "Authorization": self.get_auth_token(),
+                                            "Content-Type": "application/json"
+                                        }, get_user_agent_header()))
+            response = json.loads(request.data.decode("utf8"))
+            if isinstance(response, list) and len(response) > 0:
```

Comment:
```suggestion
            if isinstance(response, list) and response:
```
no need to check the length here